### PR TITLE
New features for PDF and Excel formats

### PR DESCRIPTION
**Pull request by arturoojeda**
PDF:
- Support for logo image on top of PDF
- Changed header row to black background and white font

Excel:
- New parameter for column autosize
- New parameter for Zebra Style on rows and header
- New parameter for titles at the top of the document

### CODE REVIEW

**Comment by arturoojeda** [comment]
Hey Nathan, I would like to update the docs too, is that something I can do or do you have to do it? I can send you the document

**Comment by nwwells** [comment]
Unfortunately, I don't own the plugin on http://grails.org/plugins/, which
is where the documentation is. I created the github project because I had
to maintain a branch while I waited for my changes to be released.

Anyway, I'll take a look at your pull request in a couple days. For your
information, Andreas Schmitt (andreas.schmitt.mi@gmail.com) is the one who
"owns" the plugin according to the Grails website.

Nathan Wells

**Comment by arturoojeda** [comment]
Got it, just sent an email to Andreas hoping he will give me a way to update it on the main Grails repo

File: ExportGrailsPlugin.groovy [hunk 1]
```diff
@@ -2,7 +2,7 @@ import org.apache.commons.logging.LogFactory
 
 class ExportGrailsPlugin {
     // the plugin version
-    def version = "1.5"
+    def version = "1.6"
```
Comment (nwwells) [inline ExportGrailsPlugin.groovy]: Since the version in the Grails plugin repo is 1.5, I suggest we keep this value the same. @arturoojeda

**Comment by nwwells** [comment]
@arturoojeda everything looks good except for the version number changes. can you modify that and I'll go ahead and merge?

**Comment by arturoojeda** [comment]
@nwwells Sure, I'll push the change in a moment. Thanks!

**Comment by arturoojeda** [comment]
@nwwells Done

**Pull request merged**